assignment/index.html

```diff
@@ -58,25 +58,49 @@
 
       /* =====================
 
-      Start code
+      Start c
 
       ===================== */
 
       var jsonToCsv = function(json) { console.log(json); };
-
-
-      var addMarkers = function(map) {};
-
-      /* =====================
-
-      End code
-
-      ===================== */
-
-
-      jsonToCsv(healthCenters);
-      addMarkers(map);
-    </script>
-    <!--Your code ends here-->
-  </body>
-</html>
+        let HospitalLocation = [['LNG','LAT','OBJECTID','NAME','ORGANIZATION','ZIP','PHONE','WEBSITE','DENTAL_PHONE','FULL_ADDRESS']]
+        for (var i=1; i<healthCenters.length; i++) {
+            var GetArray = [parseFloat(`${healthCenters[i].LNG}`),
+                     parseFloat(`${healthCenters[i].LAT}`),
+                        `${healthCenters[i].OBJECTID}`,
+                        `${healthCenters[i].NAME}`,
+                        `${healthCenters[i].ORGANIZATION}`,
+                     Number(`${healthCenters[i].ZIP}`),
```

Comment:
We encourage working together.
But I have seen this exact same strategy a few times and it's definitely not quite the right way of solving it, and I just want to make sure you are all learning.